python_modules/libraries/dagster-dg/dagster_dg/cli/plus/deploy.py

```diff
@@ -275,6 +283,8 @@ def build_and_push_command(
 @dg_global_options
 @cli_telemetry_wrapper
 def set_build_output_command(image_tag: str, **global_options: object) -> None:
+    from dagster_cloud_cli.commands.ci import set_build_output
```

Comment:
how come they need to be inline? dagster_cloud_cli shouldn't bring in the world, it just depends on dagster-shared now

Reply:
mainly just to get test mocks working without pulling teeth (otherwise these imports happen before we can mock them in the test suite)